src/protocols/rdp/rdp_settings.c

```diff
@@ -775,6 +783,14 @@ guac_rdp_settings* guac_rdp_parse_args(guac_user* user,
     settings->sftp_directory =
         guac_user_parse_args_string(user, GUAC_RDP_CLIENT_ARGS, argv,
                 IDX_SFTP_DIRECTORY, NULL);
+
+    /* Default keepalive value */
+    settings->sftp_keepalive =
+        guac_user_parse_args_int(user, GUAC_RDP_CLIENT_ARGS, argv,
+                IDX_SFTP_KEEPALIVE, 0);
+    if (settings->sftp_keepalive == 1)
```

Comment:
So, I made these unsigned instead of plain int variables, which should help eliminate the negative values.  However, I'm wondering if something needs to be done client-side to prevent negative values??